### PR TITLE
Fix break-on-load to make breakpoints on attach work

### PR DESCRIPTION
**Pull request by digeff**
* Move this._breakOnLoadHelper to common args so break on load can be used for both launch and attach
* Update the logic of getUrlRegexForBreakOnLoad to handle case insensitive paths
  * Pass caseSensitivePaths around which is an argument of the case insensitive path logic

This as final bug-fixes. At this moment we are using this change as a temporary-workaround to fix the NTVS unit tests

### CODE REVIEW

**Comment by digeff** [comment]
@rakatyal PTAL

**Comment by rakatyal** [comment]
@roblourens: Yes I believe this is still valid but we are not planning to ship this at the moment. I will review this soon.

**Comment by digeff** [comment]
@roblourens @rakatyal We are planning on using this PR for VS 16. Could you please review it now?

**Comment by roblourens** [comment]
@rakatyal are you going to review this?

**Comment by rakatyal** [comment]
@roblourens: Yes I will do this today. Apologies for missing this earlier.

File: test/chrome/chromeDebugAdapter.test.ts [hunk 1]
```diff
@@ -860,6 +859,18 @@ suite('ChromeDebugAdapter', () => {
         });
     });
 
+    suite('break-on-load', () => {
+        test('is active when the parameter is specified and we are launching', async () => {
+            await chromeDebugAdapter.launch({breakOnLoadStrategy: 'regex'});
+            assert(chromeDebugAdapter.breakOnLoadActive, 'Break on load should we active if we pass the proper parameter and we are attaching');
```
Comment (rakatyal) [inline test/chrome/chromeDebugAdapter.test.ts]: should "be" active

File: test/chrome/chromeDebugAdapter.test.ts [hunk 2]
```diff
@@ -860,6 +859,18 @@ suite('ChromeDebugAdapter', () => {
...
+        });
+
+        test('is active when the parameter is specified and we are attaching', async () => {
+            await chromeDebugAdapter.attach({breakOnLoadStrategy: 'regex', port: ATTACH_SUCCESS_PORT});
+            assert(chromeDebugAdapter.breakOnLoadActive, 'Break on load should we active if we pass the proper parameter and we are attaching');
```
Comment (rakatyal) [inline test/chrome/chromeDebugAdapter.test.ts]: should "be" active

**Pull request merged**